corehq/form_processor/submission_post.py

```diff
@@ -152,6 +153,9 @@ def _get_success_message(self, instance, cases=None):
         '''
 
         if not instance.metadata or instance.metadata.deviceID != FORMPLAYER_DEVICE_ID:
```

Comment:
There are two conditions here but I think you only want the first one for the metric (`if not instance.metadata`) otherwise this will get sent for all non-formplayer submissions.

Reply:
thanks for pointing that out!